Random Probability Maker/README.md

```diff
@@ -0,0 +1,21 @@
+
+#  Random Probability Maker
+this app takes your tasks as input and randomly selects one task for you to do .
+
+## 🚀 About Me
+I'm a full stack developer...you can follow me here -https://github.com/RAHULBAWA777
+
+
+## Run Locally
+
+Clone the project
+
+
+Start the live server
+
+
+## Used By
+
+This project is used by the following companies:
+
+- https://github.com/khushi-purwar/WebDev-ProjectKart
```

Comment:
add demo video in readme as I said earlier

Reply:
@Mrjoy832 added video